libnite2/CMakeLists.txt

```diff
@@ -75,8 +75,13 @@ install(TARGETS NiTE2
 	RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
 
-install(DIRECTORY common/include
-	DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION}
+#install(DIRECTORY common/include
+#	DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION}
+#)
```

Comment:
I changed it. The new code is written directly below, without comments. I just left the old code because I was not sure whether the new one or the old one is correct. If you tell me which is right, I will just delete the other.

So do we take option 1 or 2:
1.
install(DIRECTORY common/include
	DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})

2.
install(DIRECTORY common/include/${PROJECT_NAME}/
 	DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
 	FILES_MATCHING PATTERN "*.h"
 	PATTERN ".svn" EXCLUDE
  )